starters/basic-starter/components/media--image.tsx

```diff
@@ -41,6 +41,7 @@ export function MediaImage({
         alt={image.resourceIdObjMeta.alt || 'Image'}
         title={image.resourceIdObjMeta.title}
         priority={priority}
+        sizes="50vw"
```

Comment:
Instead of hardcoding a value here, we should allow anyone using `media--image` to specify this